graphcanvas/tests/test_graph_container.py

```diff
@@ -3,10 +3,13 @@
 import mock
 import networkx
 
+from kiva.testing import KivaTestAssistant
+
 from graphcanvas.graph_container import GraphContainer
 from graphcanvas.graph_node_component import GraphNodeComponent
 from graphcanvas.graph_view import graph_from_dict
-from kiva.testing import KivaTestAssistant
+import graphcanvas
```

Comment:
I thought it would be necessary for the `mock.patch`es below, but you are right it is extraneous.